examples/configuration.nix

```diff
@@ -42,6 +42,7 @@
   # offers outgoing functionality, but doesn't announce the Tor Hidden Service
   # under which peers can reach us.
   # services.clightning.announce-tor = true;
+  # services.clightning.plugins = with pkgs.nix-bitcoin.clightning-plugins; [ rebalance ];
```

Comment:
Are the plugins configured with clightning extraConfig? Perhaps would be helpful to mention this here.

Reply:
I wonder if there's no simpler way for the user to figure out which plugins are availabe, we should mention which plugins are available.